src/components/avatarChannel/avatarChannel.d.ts

```diff
@@ -0,0 +1,9 @@
+import React from "react"
+export interface IAvatar {
+    size?: string,
+    src?: any,
+    alt?: string,
+    className?: string,
+    disabled?: boolean,    
+    onChange: () => void
```

Comment:
Seria bom adicionar uma props style, é sempre bom pra quando for preciso fazer uma alteração direto por css.